### DISCUSSION TITLE
Support Macports

### DISCUSSION THREAD

**Discussion by esafak** [raphamorim/rio#294] [Ideas]
Macports is a popular package manager for MacOS. I see that you support homebrew. Adding macports would increase your distribution.

**Comment by raphamorim** [comment]
Hey @esafak thanks for the issue.

I agree, do you know how the process works for add rio in macports? Is this something you would like to contribute?

**Comment by esafak** [comment]
Yes, I do. Unfortunately, I do not have the time and I barely use rio at the moment so this is more for you and your users. You just need to create a PR on GitHub. [Here](https://github.com/macports/macports-ports/blob/master/www/blades/Portfile) is a cargo-based example you can copy. The salient parts are [L7 (github.setup)](https://github.com/macports/macports-ports/blob/master/www/blades/Portfile#L7) and [the checksums (L35-38)](https://github.com/macports/macports-ports/blob/master/www/blades/Portfile#L35-L38).

**Comment by raphamorim** [comment]
Macports support has been added
